src/main.h

```diff
@@ -107,7 +107,7 @@ static const unsigned int AVG_FEEFILTER_BROADCAST_INTERVAL = 10 * 60;
 /** Maximum feefilter broadcast delay after significant change. */
 static const unsigned int MAX_FEEFILTER_CHANGE_DELAY = 5 * 60;
 /** Block download timeout base, expressed in millionths of the block interval (i.e. 20 min) */
```

Comment:
Bah, this why you never add the actual values into the comments :)
Wlll update.